src/Components/TimerHeader.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { CiTimer } from 'react-icons/ci'
+import { SlArrowDown } from 'react-icons/sl'
+
+function TimerHeader (props) {
+  return (
+    <div className="flex flex-row space-x-20 text-sm border-b-4 border-grey-200 pb-2 mb-5">
+      <button className=""><SlArrowDown></SlArrowDown></button>
```

Comment:
same as < BsFillCheckSquareFill />

Reply:
We may also have to alter the < button> wrapping component, but Ill have to take a look myself.